styles/designs/carnival/parts/_module-components.scss

```diff
@@ -14,3 +16,12 @@ $ModuleWithIndentation__Para: (
         padding-left: h-spacing(2),
     ),
 );
+
+$ModuleWithIndentation__Para--Polish: (
+    _name: "ModuleWithIndentationPolishPara",
+    _subselector: " > p",
+    _properties:(
+        text-indent: h-spacing(2),
+        padding-left: h-spacing(-2),
+    ),
+);
```

Comment:
The paragraph component has the same selector and settings as the regular from above, maybe can just use the existing one?

Reply:
@wilkus27 I'm sorry I didn't saw that the spacing is inverted for Polish component. Please ignore my previous comment. Maybe it will looks nice if the `--Polish` in the components name will be changed to  `--ForPolishBook`, `PolishBooks` or just `--Inverted`. What do you think?

Reply:
I like `--Inverted` :wink: It's more descriptive.